packages/conventional-commit-lint/src/local.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright 2019 Google LLC. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { GCFBootstrapper } from 'gcf-utils';
+import appFn from './conventional-commit-lint';
+import express from 'express';
+import { config } from 'dotenv';
+import { resolve } from 'path';
+import * as bodyParser from 'body-parser';
+
+const out = config({ path: resolve(__dirname, '../../.env') });
+console.log(out);
```

Comment:
Will this env config have keys in it that shouldn't be output to logs?

Reply:
The .env in this case just has environment variables like GCP Project name, Bucket Name, Function Name etc.

```
DRIFT_PRO_BUCKET=YOUR_BUCKET_NAME
GCF_SHORT_FUNCTION_NAME=YOUR_FUNCTION_NAME
PROJECT_ID=YOUR_PROJECT_NAME
KEY_LOCATION=global
KEY_RING=YOUR_KEY_NAME
```